keyboards/inett_studio/sq80/sq80.c

```diff
@@ -0,0 +1,28 @@
+#include "sq80.h"
+#include QMK_KEYBOARD_H
+
+void matrix_init_kb(void) {
+  // Keyboard start-up code goes here
+  // Runs once when the firmware starts up
+  matrix_init_user();
+  led_init_ports();
+};
+
+void led_init_ports(void) {
+	setPinOutput(D0);
+	setPinOutput(D1);
+	setPinOutput(D2);
+	writePinHigh(D0);
+	writePinHigh(D1);
+	writePinHigh(D2);
```

Comment:
Just a heads up, you shouldn't need these, as the `led_update_kb` should be called shortly after the matrix init.

Reply:
IMO it's good to set them here, otherwise you get a brief flash on powerup.